### ISSUE TITLE
pycom.nvs_get() does not return None if non-existant key

### ISSUE THREAD

**Issue by philwilkinson40** [pycom/pycom-micropython-sigfox#306]
LoPy4 
On the latest firmware
Pycom MicroPython 1.18.2.r7 [v1.8.6-849-df9f237] on 2019-05-14; LoPy4 with ESP32
```
>>> import pycom
>>> pycom.nvs_get('anything')==None
ValueError: No matching object for the provided key
```

so I used a previous version of the firmware
Pycom MicroPython 1.18.2.r2 [v1.8.6-849-5cf02dc] on 2019-03-07; LoPy4 with ESP32

```
>>>import pycom
>>>pycom.nvs_get('anything')==None
True
>>>
```

[docs state ](https://docs.pycom.io/firmwareapi/pycom/pycom.html#pycomnvsgetkey)
If a non-existing key is given the returned value will be None.

**Comment by iwahdan88** [comment]
@philwilkinson40 yes, I confirm there is a discrepancy between the doc and the output of that API which was updated last release to support saving strings in NVS... This FW will be updated in next release
